packages/fxa-settings/src/components/App/index.tsx

```diff
@@ -55,6 +57,8 @@ export const App = ({
               <ConfirmResetPassword path="/confirm_reset_password/*" />
               <CompleteResetPassword path="/complete_reset_password/*" />
               <AccountRecoveryConfirmKey path="/account_recovery_confirm_key/*" />
+
+              <SigninReported path="/signin_reported" />
```

Comment:
The other `path`s have a  trailing `/*` at the end.

Should this be `<SigninReported path="/signin_reported" />`
or `<SigninReported path="/signin_reported/*" />`

Reply:
Oh good catch, I'll fix this